.github/workflows/ci.yml

```diff
@@ -59,15 +64,39 @@ jobs:
       # cross tests are currently broken vor armv7 and aarch64
       # see https://github.com/cross-rs/cross/issues/1311.  So on
       # those platforms we only build but do not run tests.
-      if: matrix.target != 'i686-unknown-linux-gnu'
       run: cross build --all --target ${{ matrix.target }}
       env:
         RUST_LOG: ${{ runner.debug && 'TRACE' || 'DEBUG'}}
 
+  cross_test:
```

Comment:
does splitting this off speed up things?  Because this way the compiling is done twice in two different jobs that don't share build caches, so intuitively I'd expect it to be slower as we don't have enough runners to parallelise enough to make this worth it.

Reply:
I split it up because one is just building, and the other is testing

Reply:
but once they both work maybe it should be merged back into one?

Reply:
yes, but that won't happen soon I think